pfcon/pfcon.py

```diff
@@ -44,7 +45,14 @@
         'version':              'undefined',
         'coordBlockSeconds':    10
     },
-
+    "swift": {
+        "auth_url":                 "http://swift_service:8080/auth/v1.0",
+        "username":                 "chris:chris1234",
```

Comment:
Do you really want to put this in pfcon.py and not require it to come from config?

Reply:
I initially wanted to, but that would have meant an extra "step" in the e2e setup -- i.e. a step where once pfcon is up, that it is configured. I guess I could add this logic to CUBE -- CUBE knows about swift and has these parameters. It also has the curl client pfurl so in theory CUBE could configure pfcon.

Maybe this is a better idea.